review Story/#1 initial architecture template

mthielcke opened the pull request.

Commit:
#1: Added MachineApplication component decomposition.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Added risks.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Made document "pandoc" compatible.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Structure change preparations.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Integrated Pandoc build into Maven.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Added description of copy-resources Maven plugin.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: A

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Replaced CC-BY with CC-BY-SA license

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Corrected license in architecture document.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Reenabled license header generator.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Added ".md.html" to ".gitignore".

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: License headers now automatically included.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: All chapters exist now and have license headers.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Described automatic license headers in README.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

mthielcke commented on doc/runtime/ordering_dishes.md:
```diff
@@ -0,0 +1,79 @@
+
+## Ordering Dishes
+
+> OFT: Readers are lazy, so it is best not to assume that they read the feature description in the SRS. You should at least 
```
...at least?

Commit:
#1: Added deployment.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Added a domain model.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

kaklakariada commented on README.md:
````diff
@@ -1,10 +1,111 @@
+  '-- pom.xml                  Maven project configuration
+```
+
+**&#9888; Be careful not to accidentally edit files in the target directory. It happens to us now and then and is each time a source of annoyance since the changes are gone with the next build run.** 
````
You can mark a folder as 'derived'. Then eclipse won't search it any more.

kaklakariada reviewed (approved):
Maybe provide a generated PDF or a screenshot so that you can see the final layout of the document.

Commit:
1: Initial architecture template

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Added cross-cutting concept.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Added references to guidelines.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Rewrote links via Lua filter.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

mthielcke committed:
fixed regex in link converter

Commit:
#1: Added Quality Scenarios.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Added Tags.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Improved CSS for requirement.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Changed project structure to be more Maven-compliant.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Added link checking tips. Fixed broken links.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: HTML document can now be packaged as ZIP or TAR archives

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Optical tweaks.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Automatic version and title injection. Switched to release 1.0.0.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1

Commit:
#1: Fixed typo.

Task-Url: http://github.com/itsallcode/openfasttrace-architecture-template/issues/1
